doc/_scripts/gen_devicetree_rest.py

```diff
@@ -617,6 +617,15 @@ def print_binding_page(binding, base_names, vnd_lookup, driver_sources,dup_compa
             print(f'- {specifier} cells: {", ".join(cells)}',
                   file=string_io)
 
+    # Examples
+    print_block('''\
+    Examples
+    **********
+    ''', string_io)
+    if binding.examples:
+        for example in binding.examples:
+            print(to_code_block(example), file=string_io)
```

Comment:
add language parameter to to_code_block (defaulting to "none") and pass "dts" so that we have syntax highlighting